scn/post_upload_s3.bash

```diff
@@ -68,24 +68,38 @@ do
 		#move to HOLD location
 		
```

Comment:
If I'm remembering correctly, there was some discussion about having archive / not archive S3 DCM be a DCM-side configuration option.
- Am I remembering right?
- Do we still care?
- If both, is it still worth the incremental increase in complexity (vs "S3 DCM gives you archives w\ no in-place compute")?

Reply:
That was on the table for a while but I have a pretty strong memory that we switched to only do zips.